src/snowflake/connector/cursor.py

```diff
@@ -394,16 +406,15 @@ def close(self) -> bool | None:
         try:
             if self.is_closed():
                 return False
-
             with self._lock_canceling:
                 self.reset(closing=True)
                 self._connection = None
                 del self.messages[:]
                 return True
         except Exception:
-            pass
+            return False
```

Comment:
technically this is changing behavior from returning None to return False?

Reply:
Ahh, my bad!
```suggestion
return None
```